### DISCUSSION TITLE
Make scrollbar visible without mouse

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#13852] [Q&A]
In my touchscreen application I don't have a mouse, so I can't hover over. The issue is that any time I have a scrollbar (`ScrollView`) then the scroll bar isn't visible, so the user doesn't know that there are more items available. How can I force the scroll bars to always be visible, instead of only when hover-over?

**Comment by jeromelaban** [comment]
This is generally done by setting the following on the `ScrollViewer`:

```xml
ScrollViewer.HorizontalScrollBarVisibility="Visible"
ScrollViewer.VerticalScrollBarVisibility="Visible"
```

Depending on the context though, it may not be applied. Is it for a `ListView` or something else?

**Reply by HakanL** [reply]
It still doesn't show up until you mouse over the area (which I can't, because I don't have a mouse on my touch screen device). It's a ScrollViewer hosting an ItemsRepeater.

**Reply by HakanL** [reply]
I can "simulate" it by running in on Windows (GTK/Skia) and having the mouse outside the window. It looks like this:
![image](https://github.com/unoplatform/uno/assets/407941/2ef126ba-cd19-4ecf-8294-e5c3c3cb1a5b)
And once I move the mouse over to the application then it looks like this:
![image](https://github.com/unoplatform/uno/assets/407941/a38ffc75-dc2a-4e82-a187-10a260c2a9c7)

**Reply by HakanL** [reply]
@jeromelaban Any idea on how I can workaround this? I'm on Uno5 now (same issue though, but that's expected). I basically just want the scrollbar to always be visible (since I don't have a mouse in my touch screen application/hardware).

**Reply by jeromelaban** [reply]
At this point, no, but it looks like the issue really is that:
```
ScrollViewer.HorizontalScrollBarVisibility="Visible"
ScrollViewer.VerticalScrollBarVisibility="Visible"
```
Should be enough to get the scrollbars to show, but that does not seem to be working properly for skia. Let's track in https://github.com/unoplatform/uno/issues/14474.
